src/test/java/com/productdock/library/search/book/BookMapperShould.java

```diff
@@ -21,27 +25,86 @@ class BookMapperShould {
     @Test
     void mapBookDocumentToBookDto() {
         var bookDocument = defaultBookDocumentBuilder()
-                .id("123").title("Book title").author("Book author").cover("Book cover").build();
+                .bookId("123").title("Book title").author("Book author").cover("Book cover").build();
+        List<Record> records = new ArrayList<>();
```

Comment:
Setting up test object (BookDocument in this case) should be cleaner. Try using Builder pattern to set up entire BookDocument.